### DISCUSSION TITLE
MAUI Screenshot on Android

### DISCUSSION THREAD

**Discussion by iamadamreed** [dotnet/maui#7707] [General]
The documentation makes note that the screenshot only works for the active application (https://docs.microsoft.com/en-us/dotnet/maui/platform-integration/device-media/screenshot), what would be the best practice for taking a global screenshot of the device using MAUI. 

Thanks.

**Comment by xtuzy** [comment]
Sorry, Do you ask get device screenshot after exit you app? If  that is your problem, you can google like [55](https://www.google.com/search?q=ScreenShot+after+exit+app+android), convert java code to c# very easy
